# origin: Debian (herbert)
# cset: n/a
# inclusion: not submitted
# description: forward declaration of net_device
# revision date: 2004-09-03

diff -urN kernel-source-2.4.26/include/linux/skbuff.h kernel-source-2.4.26-1/include/linux/skbuff.h
--- kernel-source-2.4.26/include/linux/skbuff.h	2004-04-14 23:05:40.000000000 +1000
+++ kernel-source-2.4.26-1/include/linux/skbuff.h	2004-04-17 14:24:06.000000000 +1000
@@ -126,6 +126,8 @@
 	skb_frag_t	frags[MAX_SKB_FRAGS];
 };
 
+struct net_device;
+
 struct sk_buff {
 	/* These two members must be first. */
 	struct sk_buff	* next;			/* Next buffer in list 				*/

